src/components/UserAvatar/index.jsx

```diff
@@ -0,0 +1,21 @@
+import PropTypes from 'prop-types';
+import SUserAvatar from './style';
+import user from '../../assets/Img/users/e3.png';
```

Comment:
FYI : Loic set up this repo to custom the imports so you can remove the ../.. ;)